addon/webextension/selector/ui.js

```diff
@@ -619,25 +601,25 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       } else {
         this.el.classList.remove("left-selection");
       }
-      this.el.style.top = (pos.top - bodyRect.top) + "px";
-      this.el.style.left = (pos.left - bodyRect.left) + "px";
-      this.el.style.height = (pos.bottom - pos.top - bodyRect.top) + "px";
-      this.el.style.width = (pos.right - pos.left - bodyRect.left) + "px";
+      this.el.style.top = `${pos.top}px`;
+      this.el.style.left = `${pos.left}px`;
+      this.el.style.height = (pos.bottom - pos.top) + "px";
```

Comment:
For consistency I suppose these should be converted too. You can include any expression in a string substitution (i.e., `${pos.bottom - pos.top}px`)

Reply:
(huh not sure why I didn't receive any notification on that comment.)

I left those because I thought using parens for calculations is a little easier to read.  I'll change them to interpolated strings for consistency.